console/src/components/CompactAppList/CompactAppList.js

```diff
@@ -0,0 +1,36 @@
+import kind from '@enact/core/kind';
+import {Row} from '@enact/ui/Layout';
+import React from 'react';
+import PropTypes from 'prop-types';
+import AppIconCell from '../AppIconCell';
+
+const CompactAppList = kind({
+	name: 'CompactAppList',
+
+	propTypes: {
+		onSelect: PropTypes.func
+	},
+
+	handlers: {
+		onTabChange: (ev, {onSelect}) => {
+			if ((ev.keyCode === 13 || ev.type === 'click') && ev.currentTarget.dataset.tabindex) {
+				onSelect({index: parseInt(ev.currentTarget.dataset.tabindex)});
+			}
+		}
+	},
+
+	render: ({onTabChange}) => (
+		<React.Fragment>
```

Comment:
While I appreciate the use of React.Fragment, without the `{...rest}` being included in the component, consumers of this component lose the ability to add their own properties to it, whether that's `style`, `className`, `data-*` attributes, or even events. It's good practice to always include support for `rest`. The homescreen doesn't send any props in _right now_ but it would be best to _support_ that feature out-of-the-box, for future improvements.

Reply:
tbh i didn't like to pass everything and delete from that as it's built in everywhere here, but I will add rest.